src/main/java/persistence/sql/common/ColumnType.java

```diff
@@ -0,0 +1,32 @@
+package persistence.sql.common;
+
+class ColumnType {
```

Comment:
```suggestion
enum ColumnType {
```

ColumnType 은 Enum 으로 선언하는 것도 가능할것 같아서 제안해봅니다.